core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala

```diff
@@ -92,6 +92,9 @@ private[spark] class TaskSchedulerImpl(
   // CPUs to request per task
   val CPUS_PER_TASK = conf.get(config.CPUS_PER_TASK)
 
+  // Resources to request per task
+  val RESOURCES_PER_TASK = conf.getTaskResourceRequirements()
```

Comment:
It is not a simple constant. So maybe call it `resourcesPerTask`

Reply:
updated